hls4ml/model/optimizer/passes/repack_stream.py

```diff
@@ -53,3 +86,38 @@ def transform(self, model, node):
         model.replace_node(node, repack_layer)
 
         return True
+
+class BroadcastStream(OptimizerPass):
+    def match(self, node):
+        if node.__class__.__name__ == 'Merge':
+            try: 
+                inp1 = node.get_input_variable(node.inputs[0])
+                inp2 = node.get_input_variable(node.inputs[1])
+                return inp1.shape != inp2.shape
+            except:
```

Comment:
Can you remind me why does this have to be in a try/except block?

Reply:
It may just be an ugly way to ensure that `len(node.inputs) == 2`, but since `Merge` layer already [asserts this](https://github.com/fastmachinelearning/hls4ml/blob/master/hls4ml/model/hls_layers.py#L1412), maybe this try/except block is not needed... I'll test and remove